proto/crosschain/query.proto

```diff
@@ -246,6 +251,11 @@ message QueryGetCctxRequest {
   string index = 1;
 }
 
+message QueryGetCctxByNonceRequest {
+  int64 chainID = 1;
```

Comment:
chain_id

Reply:
Good idea. This will be fixed by https://github.com/zeta-chain/node/issues/955